### PR TITLE
Fix image populate

### LINKED ISSUE #2118: Models not returning images on relations
**Informations**
- **Node.js version**: 10
- **npm version**: 6.4.0
- **Strapi version**: 3.0.0-alpha.14.2
- **Database**: MySQL 5.7.23
- **Operating system**: osx10.9 

**What is the current behavior?**

Images / Media are not being returned on relational models.

**Steps to reproduce the problem**

Create 2 models, call one products and the other brands. Have multiple images on a single product and then associate that product to a single brand (one-to-one). Try query the API for brands and the associated products images are not returned or even come back as NULL.

**What is the expected behavior?**

All media and images that are associated to the models should be returned in the response.

**Suggested solutions**

Spoken with @derrickmehaffy about this and he also has more clarity on the issue.

### PR DESCRIPTION
**Pull request by lauriejim**
**My PR is a:**
- [ ] 💥 Breaking change
- [x] 🐛 Bug fix #2118
- [ ] 💅 Enhancement
- [ ] 🚀 New feature

**Main update on the:**
- [ ] Admin
- [ ] Documentation
- [x] Framework
- [ ] Plugin

**Manual testing done on the following databases:**
- [ ] Not applicable
- [x] MongoDB
- [ ] MySQL
- [ ] Postgres

**Description:**

When you receive populated data, morph relations are omit and it's not good for media (images)
So now we auto populate morph relations. You will receive images.

### CODE REVIEW

File: packages/strapi-hook-mongoose/lib/index.js [hunk 1]
```diff
@@ -160,6 +160,28 @@ module.exports = function (strapi) {
                             } else {
                               this._mongooseOptions.populate[association.alias].path = `${association.alias}.ref`;
                             }
+                          } else {
+                            if (!this._mongooseOptions.populate) {
+                              this._mongooseOptions.populate = {};
+                            }
+
+                            // Images are not displayed in populated data.
+                            // We automaticaly populate morph relations.
```
Comment (Aurelsicoko) [inline packages/strapi-hook-mongoose/lib/index.js]: `We automatically` with two 'l'

**Comment by Aurelsicoko** [comment]
LGTM 👍

**Comment by jonmhutch7** [comment]
Hey @lauriejim 
This is pretty soon after your commit, but I updated strapi instance and also the strapi-hook-mongoose library and still am not seeing images on old or newly created relations. Will this require an entirely new database?

**Comment by lauriejim** [comment]
It's because it's actually not published on npm.
You will have to wait the `alpha.22` version (will come at the beginning of next week)

**Comment by jonmhutch7** [comment]
@lauriejim Unfortunately this didn't solve any existing issues with relations and images. Will this only apply to newly created contentTypes?

**Comment by lauriejim** [comment]
What is the request you did ?

**Comment by josepuma** [comment]
It didn't solve my issue. I still can't get images on relations

**Comment by zzmingo** [comment]
+1, it didn't solve. strapi version is 3.0.0-alpha.25

**Comment by tomsutton1984** [comment]
I'm seeing the same issues with 3.0.0-alpha.25.2

I have a content type (Company) with a media content field (logo), the files are being uploaded correctly - but the API is not returning the association, just "null".

I've tried suggestions from [#1783](https://github.com/strapi/strapi/issues/1783#issuecomment-418304556), but no luck. Happy to anyone else looking into this.

**Comment by Aurelsicoko** [comment]
@alexandrebodin I'm almost sure the work you're doing in the GraphQL plugin and relations will fix this issue... can you confirm?

**Comment by alexandrebodin** [comment]
@Aurelsicoko Need to be tested but I think so

**Comment by soonzhi** [comment]
i am facing this too. any updates?

**Comment by hieuxinhe94** [comment]
This issue still today 2021?

**Comment by pedrosantanna** [comment]
I'm having this issue too :( any updates?

**Comment by georgiod9** [comment]
How is there no universal solution to that? It's been 3 yrs and no one posted a proper fix yet?

I'm kind of in the same situation, my project is complete yet in production, the media uploads don't persist and the response from strapi api doesn't return relational media.

`?populate=*` is not the solution as it only populates content media and not media from a relation.

Anyone has a fix?

**Comment by Sparkah** [comment]
Another year and the same issue here.

**Pull request merged**